packages/ui-test-queries/src/utils/queryResult.ts

```diff
@@ -74,21 +79,34 @@ async function _getQueryResult(
   }
 }
 
-function wrapQueryResult(...args) {
-  return bindElementToUtilities(...args)
+// TODO remove any return type
+function wrapQueryResult<T>(
+  results: T[],
+  customMethods?: Record<string, GenericFunction>
+): any {
```

Comment:
again, this breaks type autocomplete, because it'd be buggy because of the weird bind